conans/client/build/meson.py

```diff
@@ -118,6 +118,44 @@ def _get_dirs(self, source_folder, build_folder, source_dir, build_dir, cache_bu
     def flags(self):
         return defs_to_string(self.options)
 
+    def _configure_cross_compile(self, cross_filename, environ_append):
+        cpu_translate = {
+            'armv8': ('aarch64', 'aarch64', 'little'),
```

Comment:
I'd love to see more mapping of conan/meson supported architectures
https://mesonbuild.com/Reference-tables.html#cpu-families